src/template/projects/VS2022/project_name/project_name.vcxproj

```diff
@@ -348,10 +348,6 @@
       <Project>{e89d61ac-55de-4482-afd4-df7242ebc859}</Project>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
-    <!--Additional Include Items-->
```

Comment:
The group and the comment must not be deleted from template because they are used as the entry point to replace with required headers.